src/test/scala/org/broadinstitute/dsde/firecloud/service/UserApiServiceSpec.scala

```diff
@@ -543,6 +543,130 @@ class UserApiServiceSpec extends BaseServiceSpec with RegisterApiService with Us
       }
     }
 
+    "when calling /me?userDetailsOnly=true and sam says ldap is enabled" - {
+      "OK response is returned" in {
+
+        samServer.clear(request.withMethod("GET").withPath(UserApiService.samRegisterUserV2Path))
+        samServer
+          .when(request.withMethod("GET").withPath(UserApiService.samRegisterUserV2Path))
+          .respond(
+            org.mockserver.model.HttpResponse.response()
+              .withBody("""{"userSubjectId": "1111111111", "userEmail": "no@nope.org", "enabled": true}""")
+              .withHeaders(MockUtils.header).withStatusCode(OK.intValue)
+          )
```

Comment:
There are different style preferences in the tests about copy/pasting mocking setup statements like these between tests.  I prefer to treat your testing code as you would production code and keep it DRY and extract the copy/pasted stuff out into its own method.  Do what you want here.